src/urlConvertor.ts

```diff
@@ -4,19 +4,34 @@ import * as leaflet from 'leaflet';
 import { PluginSettings } from 'src/settings';
 import * as utils from 'src/utils';
 
+/** A class to convert a string (usually a URL) into coordinate format */
 export class UrlConvertor {
 	private settings: PluginSettings;
 
+	/**
+	 * construct an instance of CoordinateParser
+	 * @param app The obsidian App instance
+	 * @param settings The plugin settings
+	 */
 	constructor(app: App, settings: PluginSettings) {
 		this.settings = settings;
 	}
 
+	/**
+	 * Parse the line where the cursor is in the editor
```

Comment:
"Check if the current editor contains a known URL that can be parsed"

Reply:
This function doesn't just check if it can be parsed. It actually parses it and returns the leaflet object.